templates/corporate/hello.html

```diff
@@ -1,562 +1,385 @@
-                    <a href="{{ apps_page_url }}linux">
-                        <i class="fa fa-linux platform-icon" aria-hidden="true"></i>
+            <div class='appshot-1'>
+                <picture>
+                    <!-- dark theme is only with webp images, since webp support was at the same time or earlier than prefers-color-scheme https://caniuse.com/?search=prefers-color-scheme https://caniuse.com/webp -->
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-dark-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-dark-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-dark-3x.webp') }} 3x"
+                      media="(min-width: 941px) and (prefers-color-scheme: dark)"/>
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-dark-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-dark-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-dark-3x.webp') }} 3x"
+                      media="(max-width: 940px) and (prefers-color-scheme: dark)"/>
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-3x.webp') }} 3x"
+                      media="(min-width: 941px)"/>
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-3x.webp') }} 3x"
+                      media="(max-width: 940px)"/>
+                    <source class='appshot-1__img'
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-1x.jpg') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-2x.jpg') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-3x.jpg') }} 3x"
+                      media="(max-width: 940px)"/>
+                    <img alt="" class='appshot-1__img' src="{{ static('images/landing-page/hello/screen-1-1x.jpg') }}"
+                      srcset="{{ static('images/landing-page/hello/screen-1-1x.jpg') }},
+                      {{ static('images/landing-page/hello/screen-1-2x.jpg') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-3x.jpg') }} 3x"/>
+                </picture>
```

Comment:
`du -shc static/images/landing-page/hello/screen*` finds this is adding 5.7MB of images, which is quite significant compared to the total size of the codebase (and carries a real cost). Do we actually need all of these sizes? If so, it's probably bearable as a cost, but we should probably have a conversation about what version of the screenshots we want to do so that we don't have to swap them out repeatedly -- the current draft in this PR has enough differences from the current app that we might not want this precise version to go live on zulip.com in the next week.